### ISSUE TITLE
4.19 Fails To Compile

### ISSUE THREAD

**Issue by tbjbu2** [davevill/UE4-Steamworks#3]
Please update this, I would love to use the microtransactions feature.

**Comment by juaxix** [comment]
I dont think that particular feature is implemented yet, no references are in the actual code
